### DISCUSSION TITLE
Doorbell servise that triggers homepod doorbell

### DISCUSSION THREAD

**Discussion by APIL0T** [RavenSystem/esp-homekit-devices#1602] [Q&A]
Please help. How to configure stateless button to trigger homepod native doorbell ? (like it works with homekit approved devices like logitech circle doorbell ) . I have devise with dry contact, plugged in ground and gpio 14 on Node MCU .

**Comment by LachNed** [comment]
Same question...
Is there a special JSON command needed to activate this feature?

**Reply by mariusvasss** [reply]
For sonoff mini with button on gpio 4

{"c":{"n":"DoorBell","l":13,"b":[{"g":0,"t":5}]},"a":[{"t":13,"f0":[{"g":4,"p":0,"i":1,"t":0}]}]}

**Reply by mariusvasss** [reply]
Gpio 4 are s1 and s2 marked on the device

**Comment by LachNed** [comment]
Thank you...
I already get notifications from the door bell,
but I want HomePod play a chime, when someone ring at the door. 
As described at [Apple Support](https://support.apple.com/lt-lt/guide/homepod/apde6dc8093d/homepod) 
_Use HomePod with a doorbell: If you have a HomeKit accessory that you use for a doorbell, tap Chime on HomePod, then choose which HomePod speakers should chime when someone rings the doorbell. And, if you have HomeKit Secure Video with Face Recognition turned on, HomePod will announce who’s there._

**Reply by toshibochan** [reply]
Apple stoped supporting doorbell with out video.

**Reply by mariusvasss** [reply]
My doorbell works on 15.2 with no problems. Maybe a restart will help....

**Reply by toshibochan** [reply]
You have HomePod saying have visitors in the front door?

**Reply by mariusvasss** [reply]
> You have HomePod saying have visitors in the front door?

No, i have a doorbell with a simple button and my homepode rings...

**Comment by jat80** [comment]
Set it up as an automation.  When doorbell is pressed play music on HomePod and choose a doorbell chime in Apple Music.  That’s what I’ve done and it works great.
